README.md

```diff
@@ -39,6 +39,16 @@ SELECT c.id, c.full_name
         FROM customers AS c
         WHERE c.full_name = 'John Doe';
 ```
+If you use `UseInternalServiceProvider` to resolve the services from an external dependency injection container, you also need to add the services required for applying naming conventions in Entity Framework Core:
```

Comment:
Not sure it's a good idea to add this - I'd rather we kept the documentation short and sweet, especially considering that UseInternalServiceProvider isn't recommended EF Core practice.

Reply:
I think it would be correct to briefly mention the AddEntityFrameworkNamingConventions method. And also mention that it is not recommended to use it, as is done in the ef core documentation.

In my case, when using UseInternalServiceProvider and not knowing about AddEntityFrameworkNamingConventions, I initially thought that the library was broken.

I will return the documentation to its original form and leave it up to you, but still believe that any documentation is better than none.